client/ayon_core/tools/loader/ui/product_group_dialog.py

```diff
@@ -15,13 +16,22 @@ def __init__(self, controller, parent):
         group_name_input = PlaceholderLineEdit(self)
         group_name_input.setPlaceholderText("Remain blank to ungroup..")
 
+        group_picker_btn = QtWidgets.QPushButton()
```

Comment:
All widgets should be parendet to correct parent.

Reply:
Implemented with https://github.com/ynput/ayon-core/pull/668/commits/1191e130bfcb6d954a70c43606eccda72d95bcd8